articles/active-directory/saas-apps/clearcompany-tutorial.md

```diff
@@ -74,6 +74,9 @@ Follow these steps to enable Azure AD SSO in the Azure portal.
 
     In the **Identifier** text box, type the URL:
     `https://api.clearcompany.com`
+    
+    In the Reply URL text box, tpye the URL:
```

Comment:
It would probably be nice to add `**` around the "Reply URL" field name here to match the formatting of other field names if you wouldn't mind making that adjustment.

Reply:
Good catch! Went ahead and submitted another commit with this bolded. Fixed the typo on the word "Type" as well.